### PR TITLE
Added OCP workplace

### PR DESCRIPTION
**Pull request by JunqiZhang0**
Added libvirt on openstack

### CODE REVIEW

**Comment by samvarankashyap** [comment]
needs rebase @JunqiZhang0

File: docs/source/examples/workspaces/openstack-ocp/OCP.pem [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (14rcole) [inline docs/source/examples/workspaces/openstack-ocp/OCP.pem:1]: Don't commit the private key

File: docs/source/examples/workspaces/openstack-ocp/ocp-run.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#jinja2:lstrip_blocks: True
+---
+libvirt-network:
```
Comment (14rcole) [inline docs/source/examples/workspaces/openstack-ocp/ocp-run.yml:3]: Why are you creating an openstack server and a libvirt node?

File: docs/source/examples/workspaces/openstack-ocp/OCP.pem (hunk 1 above)
Comment (JunqiZhang0) [reply docs/source/examples/workspaces/openstack-ocp/OCP.pem:1]: I'll delete this later, but for the current test, I'll keep it at this moment

File: docs/source/examples/workspaces/openstack-ocp/ocp-run.yml (hunk 2 above)
Comment (JunqiZhang0) [reply docs/source/examples/workspaces/openstack-ocp/ocp-run.yml:3]: For the libvirt testing

**Comment by JunqiZhang0** [comment]
[test]

**Comment by JunqiZhang0** [comment]
[merge]

**Pull request merged**